src/screens/WelcomeScreen/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { SafeAreaView, ScrollView, StatusBar, Text, View, useColorScheme } from 'react-native';
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+import styles from './styles';
+
+const Section: React.FC<{
+  title: string;
```

Comment:
What do you thing about separating the interface for the props?

```ts
interface SectionProps {
  title: string;
}; 

const Section: React.FC<SectionProps> = ({ children, title }) => {

  ...

}
```

Reply:
Yes, I was thinking of this as a good practice for all of us to have, and also for new people reading TypeScript code.

Reply:
yuppp totally agree on that, just not sure if the change in worth in this instance

Reply:
yes It was the boilerplate from RN CLI, just to have something to render